ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -273,14 +273,14 @@ public async Task correctlyCreates_PostMortemNotificationWithNoExistingDeathEven
             var notification = await GetSingleNotification(legacyId);
 
             // Assert
-            Assert.Single(notification.TreatmentEvents);
+            var deathEvent = notification.TreatmentEvents.Single(te => te.TreatmentEventIsDeathEvent);
```

Comment:
In a test, it's quite good to assert single, before selecting single, because if there isn't exactly one entry in the list then the test will fail with a better error message